util/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 SUDO = os.environ.get('ADX_SUDO', '')
 
-compose = ["docker-compose"]
+compose = ["docker-compose --verbose"]
```

Comment:
Do we really want to enable `--verbose` for local dc stack? I'm worried that it'll create too much noise in local dev env.

Reply:
Removed